user-guide/Standard_Apps/SRM/srm_getting_started/config_service_orchestration/Service_Orch_configuring_resource_and_pools.md

```diff
@@ -0,0 +1,237 @@
+
+1. In the Profiles module, make sure the *ResourceInputInterfaces* and *ResourceOutputInterfaces* capability parameters are correctly defined:
+
+   - *Category*: *Capability*
+   - *Type*: *Discrete*.
+   - *Discrete Type*: *Text*.
+   - Discrete values: Add a discrete value with the name of each supported input or output interface.
+
+   > [!TIP]
+   > See also:
+   >
+   > - [Creating profile parameters](xref:creating_profile_parameters)
+   > - [Configuring profile parameters](xref:Configuring_profile_parameters)
+
+1. In the Resources module, make sure these capabilities are added to each of the resources:
+
+   1. Select the resource, click *Add capability*, and select *ResourceInputInterfaces*.
+
+   1. In the dropdown box next to *ResourceInputInterfaces*, select the input interfaces supported by the resource.
+
+   1. Click *Add capability* again and select *ResourceOutputInterfaces*.
+
+   1. In the dropdown box next to *ResourceOutputInterfaces*, select the output interfaces supported by the resource.
+
+For example, for an "Encoding" virtual function with an "ASI" input and an "IP" output, the associated virtual resource will need a *ResourceInputInterfaces* capability with value "ASI" and a *ResourceOutputInterfaces* capability with value "IP".
+
+### Resource pool configuration
+
+To make sure users can select a resource from a resource pool at booking time, the pool names must be configured in one of the following ways:
+
+- Name consisting of two strings separated by a period, the first string being the virtual platform name, and the second the function name. For example: *SatelliteDownlink.Demodulating*.
+
+- Custom string. In this case, the service definition will need to reference that pool: on the relevant node of the service definition, a *Resource Pool* property must be defined, and its value must be set to the name of the pool.
+
+> [!TIP]
+> See also: [Configuring pools of resources](xref:Configuring_pools_of_resources)
+
+## Resource filtering configuration
+
+In the Booking Wizard, the candidate resources for a node in the service definition are filtered based on the capacities and capabilities of the profile instance selected for that node.
+
+### Setting up basic resource filtering
+
+1. In the Profiles module, make sure the necessary capability and capacity parameters are defined and added to the profile definition you want to use for a specific node. See [The Profiles module](xref:The_Profiles_module).
+
+<!-- TBD: May need more details - to be added later when the profile instances information is added -->
+
+   > [!NOTE]
+   > If capabilities or capacities are defined on an interface profile definition, having the same capability or capacity parameters on multiple interfaces of the same function is not supported.
+
+1. In the Services module, assign the profile definition to the node:
```

Comment:
That part is no longer valid (nothing needs to be done on Services module level )-> can be removed